dev/build-tool-support/src/main/scala/com/lightbend/lagom/dev/Reloader.scala

```diff
@@ -193,6 +214,7 @@ class Reloader(
    * - null - If nothing changed.
    */
   def reload: AnyRef = {
+    // fixme: synchronize on something that is guaranteed to be global in mavens multi classloader environment
```

Comment:
I'd say we should try to resolve all fixme/todo before merging. Either by fixing them, or creating tickets. (I think it'd be valuable to have a policy that is applicable to all contributions.)